web/frontend/package.json

```diff
@@ -22,11 +22,16 @@
   "dependencies": {
     "@material-ui/core": "^4.12.3",
     "@material-ui/icons": "^4.11.2",
+    "axios": "^0.24.0",
```

Comment:
Why we need this? Let's do not use additional libraries for base http fucnitons.

Reply:
Redesigned for fetch